src/content/docs/browser/single-page-app-monitoring/get-started/spa-v2.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Browser SPA monitoring 2.0
+tags:
+  - Browser
+  - Single page app monitoring
+translate:
+  - jp
+metaDescription: The new browser SPA monitoring capability is a streamlined way to monitor your users' experience and app performance.
+freshnessValidatedDate: 2024-03-18
+---
+
+<Callout variant="important">
+    This release is in limited preview and only available to select customers. Going forward, we plan on gradually phasing out the existing browser SPA experience.
+    </Callout> 
+
+For customers monitoring browser single page applications (SPA), we're excited to announce an overhaul of our SPA monitoring functionality, meant to address numerous customer pain points:
+
+* Unusable latest versions: Frequent conflicts with third-party libraries and unreliable interaction capture plagued the existing agent, often rendering the latest version unusable.
+* Patchwork solutions: Addressing one issue with SPA patches often introduced another, creating a frustrating cycle of fixes and regressions.
+* Third-party library conflicts: Global wrapping, particularly around `Promises`, often disrupted code functionality due to conflicts with other libraries.
+* Performance bottlenecks: Conflicts with code using timers, RAF, and promise chaining led to performance issues, ranging from slowdowns to freezes.
+
+The Pro+SPAv2 agent is designed to eliminate these problems and provide a significantly improved monitoring experience. Key changes include:
+
+* Unwrapped execution: By not wrapping core globals, Pro+SPAv2 unleashes execution speed boosts for your SPAs.
+* Aligned with soft navigation heuristics: The new agent adopts Google Chrome's soft navigation, providing more accurate interaction tracking and improved alignment with browser behavior.
+* Simplified interaction determination: Interactions are now defined as a UI event (click/keydown/submit -> route change -> DOM modification), offering a clearer and more efficient capture approach.
+* Focus on key metrics: While reported data remains largely unchanged, the new agent no longer tracks JavaScript execution or callback duration within interactions, streamlining the reported information.
+* Reduced interaction durations: Expect significantly shorter interaction durations, particularly for route changes. Initial page loads will see a slight reduction.
+* Potential event disassociation: You might observe previously associated `AjaxRequest` and `JavascriptError` events becoming disassociated from interactions, reflecting the focus on UI-driven interactions.
+* New API: The function [`newrelic.interaction({ waitForEnd: true })`](https://docs.newrelic.com/docs/browser/new-relic-browser/browser-apis/interaction/) allows you to customize when a user interaction ends.
```

Comment:
@cwli24 I think I added this 
```suggestion
* New API: The function [`newrelic.interaction({ waitForEnd: true })`](new docs link) allows you to customize when a user interaction ends.
```

Reply:
This is an existing function that's being extended for users with soft nav enabled. This sentence has enough explanation to inform them how to put in the flag and what it does along with our existing doc